installation/raspberry-pi-desktop/README.md

```diff
@@ -0,0 +1,20 @@
+# Raspberry Pi Desktop for PC and Mac
+
+The Raspberry Pi Foundation provides a Debian-based operating system image
+for x86 computers with the Raspberry Pi Desktop skin found in Raspbian.
```

Comment:
Might be worth mentioning that it boots on x64 computers too? (or is that too much of a subtle technical difference that most people won't understand?)